etc/onespan-sign-node.api.md

```diff
@@ -285,7 +392,7 @@ export interface DocumentVisibility {
 
 // @public
 export interface DocumentVisibilityConfiguration {
-    data: Nullable<Record<string, any>>;
+    data: CustomData;
     documentUid: Nullable<string>;
```

Comment:
What does the U stand for in `documentUid` ?

Reply:
I assume it stands for uuid 🤷 Don't know why exactly it's named like that

Reply:
Interesting, apparently UID is not synonymous with UUID.

> While UIDs can take a variety of forms depending on the application, a few examples include: URI, URL, UUID...

So a UUID is a type of UID 😆

Source: [unique identifier (UID)](https://internetofthingsagenda.techtarget.com/definition/unique-identifier-UID#:~:text=Examples%20of%20UIDs&text=A%20Uniform%20Resource%20Locator%20(URL,or%20entity%20on%20the%20Internet. )

Reply:
You are right! Good find

Reply:
Don't forget the definition of UUID = "Universally unique identifier". 
Sometimes we see acronyms so often that we forget and only remember its purpose 😆